Sources/Parsing/Parsers/Whitespace.swift

```diff
@@ -26,19 +26,30 @@ where
 
     @inline(__always)
     func consumeHorizontal() -> Bool {
+      // Unicode chars from `CharacterSet.whitespaces`
+      // General category Zs + \t
       switch bytes.first {
-      case .init(ascii: " "), .init(ascii: "\t"):
+      case 0x20, 0x9: // U+0020, \t (U+0009)
```

Comment:
FWIW we haven't found there to be much of a difference in using literals vs. this initializer, but if you've found to the contrary it could be worth it!